terragrunt/modules/release-distribution/lambdas/doc-router/index.js

```diff
@@ -66,10 +66,16 @@ exports.handler = (event, context, callback) => {
         return temp_redirect('https://www.rust-lang.org/learn', callback);
     }
 
-    // Forward versioned documentation as-is.
-    if (/^\/\d/.test(request.uri)) {
+    // Forward patch versioned documentation as-is.
+    if (/^(\d+)\.(\d+).(\d+)\/(.*)/.test(request.uri)) {
         return callback(null, request);
     }
+    // Include patch version 0 if minor versioned documentation is requested
+    const minor_versioned_pattern = /^\/?(\d+)\.(\d+)\/(.*)/
```

Comment:
```suggestion
    const minor_versioned_pattern = /^\/(\d+)\.(\d+)\/(.*)/
```
The first `/` is always there I think

Reply:
You are correct! It is always [required](https://www.rfc-editor.org/rfc/rfc7230#section-5.3.1) to be present.
Have edited accordingly.